### PR TITLE
added a debugger suggestion in the error list for runtime errors

### PR DESCRIPTION
**Pull request by hristoramirez**
Added the debugger suggestion from the design meeting on 5/27
fixes: microsoft/pxt-microbit#3095
pr_debugSuggest

### CODE REVIEW

**Comment by pelikhan** [comment]
Nice!

Can we simplify the wording? like “debug this problem”. “Try starting a debug session” is somewhat complicated.

**Comment by abchatra** [comment]
_"Debug this project"_ is better.  

It doesn't look clickable and looks like part of the text. 
Does clicking on the whole line launches the debugger? 
I would prefer something like link in _"Debug this project"_ which is clearly actionable.

**Comment by pelikhan** [comment]
Might be worth investigating rendering it as a button so that it is clearly an action.

**Comment by abchatra** [comment]
Discussed the button in the UI meeting and decided against it. Call of the action was better option here.

**Comment by pelikhan** [comment]
It’s just more text on top of a lot of text.

________________________________

File: webapp/src/errorList.tsx [hunk 1]
```diff
@@ -57,7 +59,7 @@ export class ErrorList extends React.Component<ErrorListProps, ErrorListState> {
                         <div className="exceptionMessage">
                             {pxt.Util.rlf(exception.exceptionMessage)}
                             <span className="debuggerSuggestion" role="button" onClick={this.props.startDebugger}>
-                                {lf('Try starting a debug session')}
+                                <a>{lf("Debug this project")}</a>
```
Comment (pelikhan) [inline webapp/src/errorList.tsx:62]: this use of <a> look wrong

**Comment by hristoramirez** [comment]
@abchatra I made the text and icon clickable because the icon by itself was too small of an area to click

**Comment by abchatra** [comment]
LGTM

File: theme/errorList.less [hunk 2]
```diff
@@ -75,11 +75,31 @@
         .exceptionMessage {
             padding-left: 1em;
             padding-top: 0.5em;
+            padding-bottom: 0.5em;
+
+            .debuggerSuggestion.blue {
```
Comment (pelikhan) [inline theme/errorList.less:80]: Why is the class called blue?

File: theme/errorList.less [hunk 3]
```diff
@@ -75,11 +75,31 @@
...
+                }
+            }
         }
 
-        .stackframe {
-            padding-left: 2em;
-            padding-top: 0.25em;
+        .ui.selection.list.exception {
+                margin-top: 0;
+            .stackframe.item {
```
Comment (pelikhan) [inline theme/errorList.less:99]: already nestled under .stackframe

File: theme/errorList.less (hunk 2 above)
Comment (hristoramirez) [reply theme/errorList.less]: It was residual from when I was trying to turn the icon and text blue, but I removed it since its unnecessary

**Comment by darzu** [comment]
Would it make sense to have a (?) info/help button? I'm not sure "Debug this project" will make sense to beginners.
@abchatra , @pelikhan

**Comment by darzu** [comment]
Also can you show what it looks like when the error text is really long and wraps to multiple lines?

**Comment by hristoramirez** [comment]
@darzu I placed a sample text that would overflow and this it looks like
runtime-over
